README.md

```diff
@@ -46,7 +46,7 @@ Each loader accepts a `loading` prop as a boolean. The loader will not render an
  
 ```js
 import React from 'react';
-import { css } from 'react-emotion';
+import { css } from '@emotion/core';
```

Comment:
if we are no longer importing from `react-emotion`, do we still need that package?

Reply:
according to the upgrade guide, i don't think we need it anymore? 

https://emotion.sh/docs/migrating-to-emotion-10

Reply:
No, you don't. Now the package name is `@emotion/core`.

Reply:
cool, can you remove it from package.json? 

Also, please make sure to open `docs/index.html` in chrome, and run `npm run watch` and make sure all the loaders render properly